### PR TITLE
OCM-8925 | test: automate id:72166 set and get config via rosacli

### PR DESCRIPTION
**Pull request by yuwang-RH**
[OCM-8925](https://issues.redhat.com//browse/OCM-8925) | test: automate id:72166 set and get config via rosacli
log : https://privatebin.corp.redhat.com/?458f7ca868bd01d0#9VYMCfxQBosZ3Z7adRv2uhecKHxsNGjaXySnXriJheTD

### CODE REVIEW

**Comment by yuwang-RH** [comment]
@xueli181114 @ciaranRoche Could you help review this PR?

File: tests/e2e/test_rosacli_config.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+				"client_secret",
+				"insecure",
+				"refresh_token",
+				"scopes",
+				"token_url",
+				"url",
+				"fedramp"}
+
+			By("Get config via rosacli")
+			originalConfig = make(map[string]string)
```
Comment (radtriste) [inline tests/e2e/test_rosacli_config.go:54]: Please setup the original config into the `BeforeEach`

File: tests/e2e/test_rosacli_config.go [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+				if key != "scopes" {
+					_, err := ocmService.SetConfig(key, value)
+					Expect(err).To(BeNil())
+				}
+			}
+		})
+
+		It("to set and get config via rosacli - [id:72166]", labels.High, labels.Runtime.OCMResources, func() {
+
+			configName = []string{
```
Comment (radtriste) [inline tests/e2e/test_rosacli_config.go:42]: This could be global to the file or ?

File: tests/e2e/test_rosacli_config.go (hunk 1 above)
Comment (yuwang-RH) [reply tests/e2e/test_rosacli_config.go:54]: Step "Get config via rosacli" is a checkpoint of 72166 and the key/value of originalConfig needs the output of that step. I will move the originalConfig init to BeforeEach.

File: tests/e2e/test_rosacli_config.go (hunk 2 above)
Comment (yuwang-RH) [reply tests/e2e/test_rosacli_config.go:42]: configName is used for "Get config via rosacli" step, move it under the step.

File: tests/e2e/test_rosacli_config.go [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+				"access_token":  "test_token",
+				"client_id":     "test_client_id",
+				"client_secret": "test_client_secret",
+				"insecure":      "true",
+				"refresh_token": "test_refresh_token",
+				"token_url":     "test_token_url",
+				"url":           "test_url",
+				"fedramp":       "true",
+			}
+			for key, value := range testingConfig {
```
Comment (xueli181114) [inline tests/e2e/test_rosacli_config.go:79]: Will the config set affect other cases run?

Comment (yuwang-RH) [reply tests/e2e/test_rosacli_config.go:79]: Yes. If some config like client_id is not recovered successfully, other rosacli command will be failed.

**Pull request merged**